wp-content/themes/core/routes/single/Single_Controller.php

```diff
@@ -77,17 +77,17 @@ public function get_featured_image_args(): array {
 		$alt_text = (string) get_post_meta( $image_id, '_wp_attachment_image_alt', true );
 
 		return [
-				Image_Controller::IMG_ID       => $image_id,
-				Image_Controller::AS_BG        => false,
-				Image_Controller::CLASSES      => [ 'c-single-featured-image', 'alignwide' ],
-				Image_Controller::SRC_SIZE     => Image_Sizes::CORE_FULL,
-				Image_Controller::SRCSET_SIZES => [
-					Image_Sizes::CORE_MOBILE,
-					Image_Sizes::CORE_FULL,
-				],
-				Image_Controller::IMG_ALT_TEXT => ! empty( $alt_text ) ? $alt_text : '',
-				Image_Controller::HTML         => ! empty( $caption ) ? '<figcaption class="item-single__featured-image-caption t-caption">' . wp_get_attachment_caption( $image_id ) . '</figcaption>' : '',
-			];
+			Image_Controller::IMG_ID       => $image_id,
+			Image_Controller::AS_BG        => false,
+			Image_Controller::CLASSES      => [ 'c-single-featured-image', 'alignwide' ],
+			Image_Controller::SRC_SIZE     => Image_Sizes::CORE_FULL,
+			Image_Controller::SRCSET_SIZES => [
+				Image_Sizes::CORE_MOBILE,
+				Image_Sizes::CORE_FULL,
+			],
+			Image_Controller::IMG_ALT_TEXT => ! empty( $alt_text ) ? $alt_text : '',
```

Comment:
```suggestion
			Image_Controller::IMG_ALT_TEXT => $alt_text,
```
Because we're casting the get_post_meta() as string above, this will already be an empty string if it returned nothing.